src/runtime/Makefile

```diff
@@ -193,6 +193,7 @@ DEFVALIDVIRTIOFSDAEMONPATHS := [\"$(DEFVIRTIOFSDAEMON)\"]
 #if value is 0, DAX is not enabled
 DEFVIRTIOFSCACHESIZE ?= 0
 DEFVIRTIOFSCACHE ?= auto
+DEFVIRTIOFSQUEUESIZE ?= 1024
```

Comment:
Is 1024 an appropriate value?

Reply:
CLH uses 1024 now： https://github.com/kata-containers/kata-containers/blob/3bb145c63ad1fe8c335d28029ee8ad8ca5f4f340/src/runtime/virtcontainers/clh.go#L1589

Reply:
1024 is the default value, AFAIR.